opensrs/opensrsapi.py

```diff
@@ -405,6 +411,8 @@ def register_domain(self, domain, purchase_period, user, user_id,
                             'Invalid syntax on domain')):
                     raise errors.InvalidDomain(e)
                 raise errors.DomainRegistrationFailure(e)
+            if e.response_code == self.CODE_CLIENT_TIMED_OUT:
+                raise errors.DomainRegistrationTimedOut(e)
```

Comment:
I was going by the response code docs from http://domains.opensrs.guide/docs/codes

> 705 Timed out, resubmit request. Client timed out waiting for the RA to respond.

I read that as meaning that OpenSRS made a request to some other resource and when that request times out they return a 705 from their API.